packages/client-python/src/rocketride/mixins/store.py

```
@@ -21,868 +21,353 @@
+    async def fs_write_string(self, path: str, text: str, encoding: str = 'utf-8') -> None:
+        """Write a string to a file."""
+        info = await self.fs_open(path, 'w')
+        handle = info['handle']
+        try:
+            await self.fs_write(handle, text.encode(encoding))
+            await self.fs_close(handle, 'w')
+        except Exception:
+            try:
+                await self.fs_close(handle, 'w')
+            except Exception:
+                pass
```

Ruff (0.15.7)

[warning] Use `contextlib.suppress(Exception)` instead of `try`-`except`-`pass`

Replace `try`-`except`-`pass` with `with contextlib.suppress(Exception): ...`

(SIM105)